backlog-set-due-date

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh -e
+
+# This script sets the due date on tickets in Redmine based on 
+# these conditions:
+# - The priority is not Low
+# - No previous due date set
+# - Not assigned to anyone
+# - Not blocked or resolved
+#
+# If you want to run this for testing there are two options
+# 1. Run the script without updating the tickets
+# dry_run=1 ./backlog-set-due-date
+#
+# 2. Like #1 but also read the tickets from a file
+# dry_run=1 ./backlog-set-due-date <issues-json-file>
+
+dry_run="${dry_run:-"0"}"
+project_id="${project_id:-18}"
+ticket_limit="${ticket_limit:-200}"
+host="${host:-"https://progress.opensuse.org"}"
+issues=$(mktemp)
+jquery='.issues | .[] | select(.priority.name!="Low" and .due_date==null and .assigned_to!=null and .status.name!="Blocked" and .status.name!="Resolved")'
```

Comment:
Seems like you misunderstood me. I just wanted to crosscheck if "Resolved" are included. Probably I was looking at a ticket that was previously open when you downloaded the issues list. Checking right now what the API returns I did not find any "Resolved" ticket showing up by default. So I think we do not need to set this. Also I just mentioned "Blocked" as one example, not necessarily complete. As "Feedback" basically means waiting for other events please also exclude "Feedback" for now.

Reply:
Yes, you are right. I added "Feedback" too.